src/configLoaders/JestConfiguration.ts

```diff
@@ -0,0 +1,60 @@
+// Grabbed from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jest/index.d.ts
```

Comment:
We should move this to definitely typed later.

Reply:
It's already there but it doesn't work when installing it unfortunately... I think the types need to be updated.

Reply:
Well, lets do that than 👍 so we can remove this later on

Reply:
This remark is still relevant. If the type defs from DT are (still) not up to date, I'd at least expect an issue for Stryker Jest runner saying we should use them as soon as they work (again).

Reply:
https://github.com/stryker-mutator/stryker-jest-runner/issues/29